lib/pbench/server/api/resources/query_apis/metadata_index/__init__.py

```diff
@@ -138,7 +138,17 @@ def get_aggregatable_fields(
                 self.get_aggregatable_fields(v, f"{prefix}{f}.", result)
         return result
 
-    def get_mappings(self, document):
+    def get_mappings(self, document: JSON):
```

Comment:
Type hit for the return value?  (I think you can use `-> JSON`, although I'm not sure that it's not returning a `dict`...).